app/src/main/kotlin/com/mapbox/vision/teaser/view/BaseTeaserActivity.kt

```diff
@@ -44,7 +43,7 @@ abstract class BaseTeaserActivity : BaseVisionActivity() {
 
     protected abstract fun destroyVisionManager()
 
-    protected abstract fun getFrameStatistics(): FrameStatistics
+    protected abstract fun getFrameStatistics(): FrameStatistics?
```

Comment:
Why is it nullable now?

Reply:
It was related to Recording mode, but I've modified it in latest branches, so there is no optional with this call in my last code